src/coreclr/tests/issues.targets

```diff
@@ -76,9 +76,6 @@
         <ExcludeList Include="$(XunitTestBinBase)/GC/Scenarios/muldimjagary/muldimjagary/*">
             <Issue>https://github.com/dotnet/runtime/issues/5933</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_40444/*">
```

Comment:
Why do we need to fix and re-enable this specific test in servicing?

Reply:
Ok, I see that people are seeing the test failing in release/5.0: https://github.com/dotnet/runtime/issues/44831#issuecomment-729352185

The test should not be running in release/5.0 in the first place because it is disabled. Why is the test running in release/5.0?

I think we should rather fix the infrastructure problem that is causing this test to run.

Reply:
This line is incorrect and didn't actually disable the test.